src/tools/align/components/results/AlignLabel.tsx

```diff
@@ -8,22 +8,38 @@ import { EntryType } from '../../../../uniprotkb/adapters/uniProtkbConverter';
 import { ParsedSequenceAndFeatures } from '../../utils/useSequenceInfo';
 
 import './styles/AlignLabel.scss';
+import { MSAInput } from '../../../components/AlignmentView';
+import { Sequence } from '../../../components/AlignWrapped';
 
 type Props = {
```

Comment:
I know it's not part of the PR, but we usually have more descriptive name for Prop types, in this case `AlignLabelProps`